scalapb/scalapb_2.13/src/test/scala/com/linecorp/armeria/server/scalapb/HelloServiceTest.scala

```diff
@@ -97,27 +99,27 @@ class HelloServiceTest {
       }
 
       override def onError(t: Throwable): Unit =
-        // Should never reach here.
+      // Should never reach here.
         throw new Error(t)
 
       override def onCompleted(): Unit = {
         assertThat(received).isTrue()
-        completed = true
+        completed.set(true)
       }
     })
 
     for (name <- names)
       request.onNext(HelloRequest(name))
     request.onCompleted()
 
-    await().untilAsserted(() => assertThat(completed).isTrue())
+    await().untilAsserted(() => assertThat(completed.get()).isTrue())
```

Comment:
wanna use `overridingErrorMessage`, but can't solve scala ambiguous type issue.

Reply:
Tested locally and `compileTestScala` was finished successfully.
That might be a bug in the Scala plugin of IntelliJ.
By the way, `AtomicBooleanAssert` seems correct to be interred.
How about using `assertThat(completed).overridingErrorMessage(...).isTrue()`?